R/slide.R

```diff
@@ -454,6 +455,18 @@ epi_slide_opt <- function(x, col_names, f, ..., before, after, ref_time_values,
       class = "epiproces__epi_slide_mean__list_not_supported"
     )
   }
+  if (!is.null(new_col_name)) {
+    cli_abort(
+      "`new_col_name` is not supported for `epi_slide_mean`",
+      class = "epiproces__epi_slide_mean__new_name_not_supported"
```

Comment:
```suggestion
      "`new_col_name` is not supported for `epi_slide_opt`",
      class = "epiprocess__epi_slide_opt__new_name_not_supported"
```

plus other "epiproces_" -> "epiprocess_" and "mean" -> "opt" things above and below